tests/test_integration.py

```diff
@@ -696,19 +696,19 @@ def test_predict_spectra(self,
         (
                 'chemprop_reaction',
                 'chemprop',
-                2.025709,
+                2.019870,
                 ['--reaction', '--data_path', os.path.join(TEST_DATA_DIR, 'reaction_regression.csv')]
         ),
         (
                 'chemprop_scaffold_split',
                 'chemprop',
-                1.890371,
+                1.907502,
                 ['--reaction', '--data_path', os.path.join(TEST_DATA_DIR, 'reaction_regression.csv'),'--split_type', 'scaffold_balanced']
         ),
         (
                 'chemprop_morgan_features_generator',
                 'chemprop',
-                2.848752,
+                2.846405,
```

Comment:
Why did these test values change?

Reply:
Because the reactions in the test suite contain unbalanced reactions if explicit_h is not specified, and some of them are on the reactant side, which is affected by the bugfix. The bug should not have influenced real data, though.